test/.clang-tidy

```diff
@@ -0,0 +1,4 @@
+InheritParentConfig: true
+CheckOptions:
+  - key: readability-identifier-naming.ClassCase
+    value: aNy_CasE
```

Comment:
Why any case here?

Reply:
There are some test records which I don't want to touch right now.
```cpp
struct memory_access_opcode
{
    evmc_opcode opcode;
    int memory_index_arg;
    int memory_size_arg;
};
```